client/www/pages/essays/[slug].tsx

```diff
@@ -20,7 +20,8 @@ function Prose({ html }: { html: string }) {
 }
 
 const Post = ({ post }: { post: Post }) => {
-  const { title, date, mdHTML, author } = post;
+  const { title, date, mdHTML, authors } = post;
+  const author = authors[0];

```

Comment:
Do you still need this `author = authors[0]`?

Reply:
oi, nice catch!